about-peer-review/how-peer-review-works.md

```diff
@@ -0,0 +1,47 @@
+# How pyOpenSci's open software peer review works
+
+
+## Who submits packages and who runs the reviews 
+
+pyOpenSci's [suite of packages](https://www.pyopensci.org/python-packages/) are 
+contributed by community members with a great diversity of skills. This diversity 
+of developer backgrounds results in a range of quality associated with the suite 
+of tools available to process scientific data.
+
+### Who reviews pyOpenSci packages ?
+
+Our peer review process is run by volunteer members of the Python scientific 
+community:
+
+* Editors manage the incoming package review submissions and ensure 
+reviews move forward progress of submissions; 
+* authors create, submit and improve their package; 
+* Reviewers, two per submission, examine the software code and user experience. 
```

Comment:
Not directly related to this file _per se_, is there a plan to maintain a list of potential reviewers based on their specific field of expertise?

Reply:
great question @Batalex we actually have a google sheet already with this type of information. i can share it in our slack with you if you want to have a look. we have people signing up regularly. what i don't love about what we have is that it is a google sheet. i'd rather have a database which is what ropensci has. but that is another discussion.

feel free to ping me with more questions about this because it's a great question and something i've been thinking about in terms of how do we manage this.